actionpack/test/dispatch/system_testing/driver_test.rb

```diff
@@ -148,7 +148,7 @@ class DriverTest < ActiveSupport::TestCase
     end
   end
 
-  test "preloads browser's driver_path" do
+  test "preloads browser's driver_path if it is set to a Proc" do
```

Comment:
If we drop `service.driver_path.try(:call)`, I think we can drop this test too, because Rails would no longer be responsible for this behavior.

Thought it might be worth adding a test for when `driver_path` is set to an actual path (to verify that it is not overwritten).

Reply:
Agreed. I'll drop the Proc test and add another test to verify an existing value is not overwritten.

Reply:
Addressed in 916efae770 and 868599937a